lib/kafka_ex/server.ex

```diff
@@ -391,7 +391,7 @@ defmodule KafkaEx.Server do
                        Metadata.parse_response(data)
                    end
 
-                   case Enum.find(response.topic_metadatas, &(&1.error_code == :leader_not_available)) do
+        case Enum.find(response.topic_metadatas, &(&1.error_code == :leader_not_available)) do
```

Comment:
No but it was incorrect before.  I just happened to be in this file debugging.  I can remove it because it's irrelevant but it should be fixed.